python/cucim/src/cucim/skimage/measure/_moments.py

```diff
@@ -519,20 +520,19 @@ def centroid(image, *, spacing=None):
 
 
 def _get_inertia_tensor_2x2_kernel():
```

Comment:
```suggestion
@cp.memoize(for_each_device=True)
def _get_inertia_tensor_2x2_kernel():
```

Reply:
I would be okay with removing it, given that it should now be <1 microsecond benefit to having it. No strong opinion either way in this case.